frontend/README.md

```diff
@@ -88,9 +88,6 @@ A common case during development is to have 3 terminals open: one for the dev se
 
     # terminal 2
     $ make test-frontend
-
-    # terminal 3
-    $ make flow
```

Comment:
Nice. There's one last mention of `flow` in the README that still needs to be removed.